### ISSUE TITLE
Inspect in workspace/debugger complains with DNU #onDblClick

### ISSUE THREAD

**Issue by paulwilke** [amber-smalltalk/helios#73]
Installed amber in version 15.1 and created a new project. Inspecting any object throws DNU. 

Bug occurs in Chrome 48.0.2564.116 (64-bit). 

The issue was first reported in https://github.com/amber-smalltalk/amber/issues/1217 and moved here because it works in legacy IDE and therefore is a Helios bug.

Adding es5 compatibility doesn't change a thing: `bower install amber-compat-es5 --save` 

Also installing Helios development directly results in the same problem:

```
git clone https://github.com/amber-smalltalk/helios.git
cd helios
npm install
bower install
grunt devel 
amber serve
```

**Comment by paulwilke** [comment]
Ok, but shouldn't I already be using this version if I installed helios development? 
I did install Helios dev (helios@0.6.0) like I described in the Bug description above.
I get the issue anyways.

**Comment by herby** [comment]
@paulwilke I don't get you.

**Comment by herby** [comment]
What I told in the comment to https://github.com/amber-smalltalk/helios/commit/e3e847e44fd1e425ef44347eaaaf5609c89ae278#diff-a4b5e41ada92c2673d80a70b7c19cfa1R726 was that 3233e94 treated one of the methods to do renderContentOn: after `wrapper :=` assignenment, but it was different method. And maybe if the one that is commented is treated similarly, the bug may be fixed. That method which is commented is still in the state shown in the commented commit, 3233e94 did not fix it.

**Comment by herby** [comment]
Edited the commit comment as well as previous comment, I was pointing to wrong commit which fixed one of similar methods, now it's proper hash: 313899a6.

**Comment by herby** [comment]
Fixed.

**Comment by herby** [comment]
Released as 0.6.1.
